staging/src/k8s.io/client-go/tools/cache/controller.go

```diff
@@ -79,8 +80,13 @@ type Config struct {
 	RetryOnError bool
 
 	// Called whenever the ListAndWatch drops the connection with an error.
+	//
+	// TODO (https://github.com/kubernetes/kubernetes/issues/126379): logcheck:context // WatchErrorHandlerWithContext should be used instead of WatchErrorHandler in code which supports contextual logging.
```

Comment:
Just curious. Are you aware of a way to detect usages of types with this comment? (e.g. like there's a linter that finds all usages of deprecated types)

Reply:
Yes, there is a linter (logcheck). https://github.com/kubernetes/kubernetes/issues/126379 explains how this is going to be handled.

Reply:
Thx! I'll try if we can use that in CR

Reply:
https://github.com/kubernetes/kubernetes/pull/126387/files#r1811006785